app/src/main/java/space/taran/arknavigator/mvp/presenter/ResourcesPresenter.kt

```diff
@@ -30,9 +32,24 @@ class ResourcesPresenter(
     @Inject
     lateinit var resourcesIndexFactory: ResourcesIndexFactory
 
+    @Inject
+    lateinit var dataStoreManager: DataStoreManager
+
     private lateinit var index: ResourcesIndex
     private lateinit var storage: TagsStorage
 
+    var sorting: Sorting = Sorting.DEFAULT
+        set(value) {
+            field = value
+            presenterScope.launch { dataStoreManager.setUserSorting(value.ordinal) }
```

Comment:
I've experimented with different ways of setting these values, and this one seemed the best, since it takes up less space, and moving away from overpopulating the class with getters/setters like it was in Java, but no prob, I'll replace these with dedicated methods.

Reply:
If @mdrlzy means separate dedicated setter, then yeah, maybe explicit function would be clearer that something complex is going on inside.

Reply:
I meant `dataStoreManager.setUserSorting` must accept `Sorting` not `Int`

Reply:
Oh. No problem. Makes total sense.